src/main/java/de/codecrafter47/taboverlay/config/dsl/CustomPlaceholderConfiguration.java

```diff
@@ -292,6 +312,7 @@ public ColorAnimation() {
         @Override
         public PlaceholderBuilder<?, ?> bindArgs(PlaceholderBuilder<Context, ?> builder, List<PlaceholderArg> args, TemplateCreationContext tcc) {
             List<TextColor> colors = new ArrayList<>();
+            List<TextColor> formats = new ArrayList<>();
```

Comment:
You could just use a string.

Reply:
But how should cases like `"BOLD, ITALIC"` and `"BOLD,ITALIC"` (Assuming we use a format similar to playerOrder) be handled?

Reply:
I was thinking `String formats` and the value should be the formatting codes, i.e. `&n&o`.

Reply:
That could work...
I think just passing the String as is without any real checks should be fine... At most a null check? Or would there be a way for `MarkedStringProperty` to default to an empty String when not set?
Would like to avoid `if (string == null) { format = ""; }` stuff...

Reply:
Another problem I see with the `&o&l` aproach is, what to do when people provide invalid formats...
Just having it pass through would unintentionally break the general format of the string and mess up the effective length you mentioned, so this would require some color validations in the end to make sure we don't allow invalid/garbage values to be provided.

Reply:
I was thinking, to use a string to pass the format to the `CustomPlaceholderColorAnimation`. That will remove some overhead and keep that class simple. Having some abstraction in the config has advantages, as you have described. So I'm good with how you designed that.